bdse-kvnode/src/main/java/ru/csc/bdse/impl/api/KeyValueStorageNodeController.java

```diff
@@ -0,0 +1,80 @@
+package ru.csc.bdse.impl.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import ru.csc.bdse.model.kv.KeyValue;
+import ru.csc.bdse.model.kv.KeyValueStorageNode;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.Properties;
+
+/**
+ * Provide HTTP API for [[KeyValueStorageNode]]
```

Comment:
Предложение должно быть в третьем лице, а не во втором.

> Use 3rd person (descriptive) not 2nd person (prescriptive). 
The description is in 3rd person declarative rather than 2nd person imperative.
Gets the label. (preferred)
Get the label. (avoid)


http://www.oracle.com/technetwork/java/index-137868.html